itests/eth_api_test.go

```diff
@@ -52,44 +60,147 @@ func TestEthAddressToFilecoinAddress(t *testing.T) {
 }
 
 func TestFilecoinAddressToEthAddress(t *testing.T) {
-	// Disable EthRPC to confirm that this method does NOT need the EthEnableRPC config set to true
-	client, _, _ := kit.EnsembleMinimal(t, kit.MockProofs(), kit.ThroughRPC(), kit.DisableEthRPC())
+	blockTime := 100 * time.Millisecond
+	client, _, ens := kit.EnsembleMinimal(t, kit.MockProofs(), kit.ThroughRPC(), kit.DisableEthRPC())
+	ens.InterconnectAll().BeginMining(blockTime)
 
 	ctx, cancel := context.WithTimeout(context.Background(), time.Minute)
 	defer cancel()
 
-	secpDelegatedKey, err := key.GenerateKey(types.KTDelegated)
-	require.NoError(t, err)
+	mkParams := func(addr address.Address, blkNum string) jsonrpc.RawParams {
+		fp := new(ethtypes.FilecoinAddressToEthAddressParams)
+		fp.FilecoinAddress = addr
 
-	filecoinKeyAddr, err := client.WalletImport(ctx, &secpDelegatedKey.KeyInfo)
-	require.NoError(t, err)
+		if blkNum != "" {
+			fp.BlkParam = &blkNum
+		}
 
-	ethAddr, err := ethtypes.EthAddressFromFilecoinAddress(filecoinKeyAddr)
-	require.NoError(t, err)
+		params, err := json.Marshal(fp)
+		require.NoError(t, err)
 
-	apiEthAddr, err := client.FilecoinAddressToEthAddress(ctx, filecoinKeyAddr)
-	require.NoError(t, err)
+		return jsonrpc.RawParams(params)
+	}
 
+	// Test for f4 address
+	filecoinDelegatedAddr, err := client.WalletNew(ctx, types.KTDelegated)
+	require.NoError(t, err)
+	ethAddr, err := ethtypes.EthAddressFromFilecoinAddress(filecoinDelegatedAddr)
+	require.NoError(t, err)
+	apiEthAddr, err := client.FilecoinAddressToEthAddress(ctx, mkParams(filecoinDelegatedAddr, ""))
```

Comment:
if you look at the other uses of `RawParams` in itests there's a use of lib/result/result.Wrap that makes this a bit easier and avoids the extra helper function:

```go
	history, err := client.EthFeeHistory(ctx, result.Wrap[jsonrpc.RawParams](
		json.Marshal([]interface{}{5, "0x10"}),
	).Assert(require.NoError))
```

Reply:
Done.